src/pages/WorkspacePage.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Navigate, useParams } from 'react-router-dom';
+import { setActiveWorkspace } from '@openshift/dynamic-plugin-sdk-utils';
+import { useWorkspace } from '../utils/workspace-context-utils';
+
+const WorkspacePage: React.FunctionComponent = () => {
+  const name = useParams()?.workspaceName;
+  const workspace = useWorkspace();
+  if (!workspace) {
+    setActiveWorkspace(name);
+  }
+
+  return <Navigate to={`/stonesoup/workspaces/${workspace}/applications`} />;
```

Comment:
I think we need to replace route here, so it doesn't go to 404 page when we try to navigate back.
```suggestion
  return <Navigate to={`/stonesoup/workspaces/${workspace}/applications`} replace />;
```